shimmingtoolbox/cli/mask.py

```diff
@@ -303,6 +304,41 @@ def sct(fname_input, fname_output, contrast, centerline, file_centerline, brain,
     return fname_output
 
 
+@mask_cli.command(context_settings=CONTEXT_SETTINGS,
+                  help="Create a mask to shim single voxel MRS. "
+                       "Voxel position and size can be directly given or these info can be read "
+                       "from the siemens raw-data. "
+                       "The mask is stored by default under the name 'mask_mrs.nii.gz' in the output "
+                       "folder. Return an output nifti file to be used as a mask for MRS shimming.")
+@click.option('-i', '--input', 'fname_input', type=click.Path(), required=True,
+              help="Input path of the fieldmap to be shimmed.")
+@click.option('-r', '--raw_data', type=click.Path(),
+              help="Input path of the of the siemens raw-data (supported extention .rda)")
```

Comment:
Why remove the twix extension? We should find a way to test for it but we should have support for both

Reply:
The standard Siemens raw data for MRS is rda file. It's more accessible to every user without the need to enable the advanced user to save it. Also as discussed before the twix files are larger (with file sizes reaching 300 MB compared to the much smaller 20 KB of RDA files) and cannot be stored on the Github testing repo.

Reply:
Indeed, but what if someone is not using Siemens.. I think we should keep the possibility of using twix files and if we can't test for it in the CI because it is too big, we can test locally.

Reply:
@behrouzvia We should add back support for twix data

Reply:
> @behrouzvia We should add back support for Twix data

Well, the twix format is also specific to Siemens, and each vendor has their own specific format for MRS. For example, Philips' format is sdat and GE uses a P-fle format.